sphinx/developing/autodoc.rst

```diff
@@ -0,0 +1,128 @@
+Autodoc
+=======
+
+The built-in extension ``sphinx.ext.autodoc`` provides functionality for
+generating API documentation from source code.
+
+Writing Docstrings
+------------------
+
+.. _Google Docstrings: https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments
+.. _Numpy Docstrings: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt
+.. _Napoleon Google Example: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
+.. _Napoleon Numpy Example: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html
+
+This documentation uses ``sphinx.ext.napoleon`` to pre-process docstrings
+allowing them to be written in plain text as opposed to ReStructuredText.
```

Comment:
'reStructructuredText' - I think it just needs to be made consistent throughout the docs (elsewhere it is 'RST' and ReStructuredText')

Reply:
there's a typo in my typo correction...